tests/data/placebo/test_efs_kms_key_filter/kms.ListKeys_1.json

```diff
@@ -0,0 +1,130 @@
+{
+    "status_code": 200,
+    "data": {
+        "Keys": [
+            {
+                "KeyId": "0c34a807-a390-4128-ba02-463ccd3d8850",
+                "KeyArn": "arn:aws:kms:us-east-1:644160558196:key/0c34a807-a390-4128-ba02-463ccd3d8850"
+            },
+            {
+                "KeyId": "13121aca-ffd6-45af-b640-4eb89f6d8fbd",
```

Comment:
can you remove some of the extraneous data in the recordings, list key outputs in particular have lots of unrelated data to the tests.

Reply:
I had copied those files from a [dynamodb](https://github.com/cloud-custodian/cloud-custodian/blob/8b9c9527f151e565d3be109586f2ca0273d61695/tests/data/placebo/test_dynamodb_kms_key_filter/kms.ListKeys_1.json) test suite.  But you're right, a cleanup was appropriate.